src/components/ebay-breadcrumb/test/test.server.js

```diff
@@ -2,64 +2,59 @@ const expect = require('chai').expect;
 const testUtils = require('../../../common/test-utils/server');
 const mock = require('../mock/index');
 
-test('shoudl return zero nav element in the page', context => {
+describe('should render with basic breadcrumb and ', () => {
+    test('return nav element with input', context => {
+        const $ = testUtils.getCheerio(context.render(mock.basicItems));
```

Comment:
You're still using the same render, so you can add a `beforeEach` to make the code simpler, and more DRY.

```
describe('should render with basic breadcrumb and ', () => {
    let $;

    beforeEach(context => {
        $ = testUtils.getCheerio(context.render(mock.basicItems));
    });

    test('return nav element with input', () => {
        ...
    });

    ...
});
````